assets/src/components/settingsPage.tsx

```diff
@@ -53,6 +57,32 @@ const SettingsPage = (): ReactElement<HTMLDivElement> => {
               { label: "Vehicle #", value: VehicleLabelSetting.VehicleNumber },
             ]}
           />
+
+          {featureIsEnabled("minischedules_trip_label") && (
+            <DropdownSetting
+              icon={mapIcon}
+              label="Trip Label"
+              selectId="minischedules-trip-label-setting"
+              value={userSettings.minischedulesTripLabel}
+              onChange={
+                // Ignoring this because we can't get the tests working and
+                // neither of can figure out why
+                /* istanbul ignore next */
+                (value) => {
```

Comment:
Given what I learned in the process of working on #1035, do you think you could use that same basic approach to get tests for this working?

Reply:
I'll give it a whirl